From 3780cef8033b5b338a36aa766b555d92b9855825 Mon Sep 17 00:00:00 2001
From: Peter Marschall <peter@adpm.de>
Date: Wed, 1 Jun 2011 21:01:23 +0200
Subject: [PATCH 09/26] opensc-tool: update EF structure names

Use EF structure names that are more sensible & grammatically more correct.

Signed-off-by: Peter Marschall <peter@adpm.de>
---
 src/tools/opensc-tool.c |   14 +++++++-------
 1 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/src/tools/opensc-tool.c b/src/tools/opensc-tool.c
index 76b4bbf..075380b 100644
--- a/src/tools/opensc-tool.c
+++ b/src/tools/opensc-tool.c
@@ -341,13 +341,13 @@ static int print_file(sc_card_t *in_card, const sc_file_t *file,
 	printf("type: %-3s, ", tmps);
 	if (file->type != SC_FILE_TYPE_DF) {
 		const id2str_t ef_type_name[] = {
-			{ SC_FILE_EF_TRANSPARENT,         "transpnt"     },
-			{ SC_FILE_EF_LINEAR_FIXED,        "linrfix"      },
-			{ SC_FILE_EF_LINEAR_FIXED_TLV,    "linrfix(TLV)" },
-			{ SC_FILE_EF_LINEAR_VARIABLE,     "linvar"       },
-			{ SC_FILE_EF_LINEAR_VARIABLE_TLV, "linvar(TLV)"  },
-			{ SC_FILE_EF_CYCLIC,              "lincyc"       },
-			{ SC_FILE_EF_CYCLIC_TLV,          "lincyc(TLV)"  },
+			{ SC_FILE_EF_TRANSPARENT,         "transparent"           },
+			{ SC_FILE_EF_LINEAR_FIXED,        "linear-fixed"          },
+			{ SC_FILE_EF_LINEAR_FIXED_TLV,    "linear-fixed (TLV)"    },
+			{ SC_FILE_EF_LINEAR_VARIABLE,     "linear-variable"       },
+			{ SC_FILE_EF_LINEAR_VARIABLE_TLV, "linear-variable (TLV)" },
+			{ SC_FILE_EF_CYCLIC,              "cyclic"                },
+			{ SC_FILE_EF_CYCLIC_TLV,          "cyclic (TLV)"          },
 			{ 0, NULL }
 		};
 		const char *ef_type = "unknown";
-- 
1.7.5.3

